portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/* nav buttons */
+
+var sections = ['.home', '.projects', '.cats', '.skills']
+
+console.log(document.querySelector('section' + sections[0]))
+
+function makeScrollCallBack(section) {
```

Comment:
Can you please add a short documentation to this function explaining what it does?